apps/accounts/verify.py

```diff
@@ -9,17 +11,27 @@
 def fxa_identify(code, config=None):
     """Get an FxA profile for an access token. If identification fails an
     IdentificationError is raised."""
-    token = get_fxa_token(code, config)['access_token']
-    return get_fxa_profile(token, config)
+    statsd.incr('accounts.fxa.identify.count.all')
+    try:
+        with statsd.timer('accounts.fxa.identify.time.all'):
+            token = get_fxa_token(code, config)['access_token']
+            profile = get_fxa_profile(token, config)
+    except:
+        statsd.incr('accounts.fxa.identify.count.fail')
```

Comment:
sounds like maybe this should be `accounts.fxa.identify.all.fail` and `accounts.fxa.identify.all.success` because you can just sum `accounts.fxa.identify.all.*` to get the total. That way you wouldn't need a separate increment of `accounts.fxa.identify.all` either.